sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/HardcodedIpAddress.vb

```diff
@@ -0,0 +1,32 @@
+﻿Imports System
+Imports System.Collections.Generic
+
+Namespace Tests.Diagnostics
+    Public Class AnyAssemblyClass
+        Public Sub New(ByVal s As String)
+        End Sub
+    End Class
+
+    Public Class HardcodedIpAddress
+        Private Shared Sub WriteAssemblyInfo(ByVal assemblyName As String, ByVal version As String, ByVal author As String, ByVal description As String, ByVal title As String)
+        End Sub
+
+        <SomeAttribute("127.0.0.1")>
+        Public Sub Foo()
+            Dim ip As String = "192.168.0.1" 'Noncompliant {{Make sure using this hardcoded IP address '192.168.0.1' is safe here.}}
```

Comment:
could you add a test with 
`Dim urlWithPort As String = "http://192.168.0.1:8000/admin.html"; // Noncompliant `

Also, from the RSPEC

> Strings of the form 2.5.<number>.<number> as they often match Object Identifiers (OID).

Reply:
The URI will be caught by [S1075](https://jira.sonarsource.com/browse/RSPEC-1075)

Reply:
I updated the compliant/noncompliant examples and the exceptions to be the same as C#